app/panel/components/Header.jsx

```diff
@@ -135,6 +135,12 @@ class Header extends React.Component {
 	}
 
 	clickLogo = () => {
+		const { entries, location } = this.props.history;
+		if (location.pathname === '/stats' &&
+		entries[entries.length - 2].pathname === '/subscription/info') {
```

Comment:
@Eden12345 can we do this in a "safer" way just in case `entries` has a length lower than 2? Could that ever be an issue?

Reply:
As the app stands it shouldn't be an issue because you have to leave the menu to get to a view where the header has a back option, but I still see your point and will add a check there.